cfme/web_ui/expression_editor.py

```diff
@@ -251,7 +251,7 @@ def fill_count(count=None, key=None, value=None):
             type="Count of",
             count=count,
             key=key,
-            value=int(value),
```

Comment:
Found this bug through PRT results, not related to the adv search changes below.